modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -45,23 +45,28 @@ spec:
      outputRefs:
      - elasticsearch-secure <9>
      - default <10>
-   - name: infrastructure-audit-logs <11>
+     labels:
+       logs: application <11>
+   - name: infrastructure-audit-logs <12>
      inputRefs:
      - infrastructure
      outputRefs:
      - elasticsearch-insecure
+     labels:
+       logs: audit-infra
 ----
 <1> The name of the `ClusterLogForwarder` CR must be `instance`.
 <2> The namespace for the log forwarding CR must be `openshift-logging`.
 <3> Specify a name for the output.
 <4> Specify the `elasticsearch` type.
 <5> Specify the URL and port of the external Elasticsearch instance as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
 <6> If using an `https` prefix, you must specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent.
-<7> Optional. Specify a name for the pipeline.
+<7> Optional: Specify a name for the pipeline.
 <8> Specify which log types should be forwarded using that pipeline: `application,` `infrastructure`, or `audit`.
 <9> Specify the output to use with that pipeline for forwarding the logs.
-<10> Optional. Specify the `default` output to send the logs to the internal Elasticsearch instance.
-<11> Optional. Configure multiple outputs to forward logs to other external Elasticsearch instances.
+<10> Optional: Specify the `default` output to send the logs to the internal Elasticsearch instance.
+<11> Optional: A label to add to the logs. 
```

Comment:
@mburke5678 we can add several labels here, so `A label` is not accurate.
The labels can be added like:
```
labels:
  name1: value1
  name2: value2
  ...
```